File: C#-Döngüler/C#-Döngüler/Form1.cs
namespace C__Döngüler
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void button1_Click(object sender, EventArgs e)
		{
			int i;
			for (i = 1; i <= 10; i++)
			{
				listBox1.Items.Add(i + "Merhaba Ankara");
			}
		}

		private void button2_Click(object sender, EventArgs e)
		{
			for (int i = 1; i < 11; i++)
			{
				listBox2.Items.Add(i);
			}
		}
	}
}

File: WinFormsApp3/WinFormsApp3/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        ListBox1.Items.Add(MaskedTextBox1.Text + " " + MaskedTextBox2.Text + " " + MaskedTextBox3.Text + " " + MaskedTextBox4.Text)
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        ListBox1.Items.Add(DateTimePicker1.Text)
    End Sub
End Class
